jdat_notebooks/README.md

```diff
@@ -98,33 +98,35 @@ The [STScI guidelines on data storage for notebooks](https://github.com/spacetel
 should be followed here.  The specific addition for the JWST Notebooks is that notebook data should be 
 in the `DMD_Managed_Data/JWST/jwst-data_analysis_tools` Box folder (or subfolders thereof).  
 If you do not have access to this box folder already, ask a Project Scientist and they should be able to get you added. 
-Note that if a draft notebook is using data that should not yet be public, the easiest choice is probably central store, 
+Note that if a baseline notebook is using data that should not yet be public, the easiest choice is probably central store, 
 but in that case it is critical that the notebook state prominently that it must be run inside the STScI network.
 3. A notebook should state clearly what version of various dependencies were used to generate the notebook. 
 These versions should be placed in a `requirements` file in the same directory as the notebook itself. An example of this file
 is in the``example_notebook`` folder.
 That will ensure reviewers/testers can be sure that if they encounter problems, it is not due to software version mis-matches.
 
+The notebook will undergo a scientific and a technical review and it will be merged into the repository
+once the review comments have been addressed. This concludes the Baseline Stage.
```

Comment:
```suggestion
once the review comments have been addressed.  At this point the notebook has formally reached the Baseline Stage.
```

Maybe this is being overly-pedantic, but I think we want to call the notebook baseline when it's *done* with this process, rather than notebooks that are currently undergoing the process being called "baseline".

Reply:
Yes, I agree with you. The wording I used was misleading. It is clearer with your edit!